english/rib-eye/rib-eye.md

```diff
@@ -14,4 +14,8 @@ or
 
 or
 
+[In Camelot] (english/camelot/camelot.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.